routes/socket/models.js

```diff
@@ -36,13 +38,16 @@ module.exports.getPowerFromRole = role => {
 	if (role === 'editor') return 2;
 	if (role === 'moderator') return 1;
 	if (role === 'altmod') return 0; // Report AEM delays will check for >= 0
+	if (role === 'trialmod') return -1;

```

Comment:
Trial mod power should be 0.

Reply:
The way the comment was worded doesn't 0 give trial mods delayed reports then? I thought according to what I read in wiki that wasn't an included power?

Reply:
Yes, trial mods have access to that feed and need to trigger the delay. The wiki is outdated.

Reply:
Ok I'll do that too wanted to make sure. Thx. Anything else so far I probably broke? First to admit still a learner

Reply:
Like a different variable name instead of staffRole ??

Reply:
Yeah like isContributor or something.

Reply:
Ok will take more time but doable with motivation #life